src/dashboard/src/components/administration/models.py

```diff
@@ -50,3 +51,4 @@ class ArchivesSpaceConfig(models.Model):
     access_conditions = models.CharField(max_length=50, blank=True, verbose_name='Conditions governing access')
     use_conditions = models.CharField(max_length=50, blank=True, verbose_name='Conditions governing use')
     repository = models.IntegerField(default=2, verbose_name='ArchivesSpace repository number')
+    inherit_notes = models.BooleanField(default=True, verbose_name='Inherit digital object notes from the parent component')
```

Comment:
I've noticed that `syncdb` populates the column as ``inherit_notes` tinyint(1) NOT NULL`, so I've updated my migration script to update the existing row so `inherit_notes` equals `True`. I guess that under normal circumstances the according Django migration would do that for me?